### PR TITLE
v.in.ogr: rename option location to project

### PR DESCRIPTION
**Pull request by petrasovaa**
Includes changes in user visible strings in v.in.ogr and v.import.

### CODE REVIEW

**Review by nilason — APPROVED** [review]

File: scripts/v.import/v.import.py [hunk 1]
```diff
@@ -93,8 +93,8 @@
 # %end
 # %flag
 # % key: o
-# % label: Override projection check (use current location's projection)
-# % description: Assume that the dataset has the same projection as the current location
+# % label: Override projection check (use current project's projection)
+# % description: Assume that the dataset has the same projection as the current project
```
Comment (veroandreo) [inline scripts/v.import/v.import.py:97]: I know the changes are targeted to location-->project, but now project's projection sounds repetitive... can we change that to coordinate reference system or CRS?
```suggestion
# % label: Override projection check (use current project's CRS)
# % description: Assume that the dataset has the same coordinate reference system (CRS) as the current project
```

File: vector/v.in.ogr/main.c [hunk 2]
```diff
@@ -377,9 +377,9 @@ int main(int argc, char *argv[])
     flag.over = G_define_flag();
     flag.over->key = 'o';
     flag.over->label =
-        _("Override projection check (use current location's projection)");
+        _("Override projection check (use current project's projection)");
```
Comment (veroandreo) [inline vector/v.in.ogr/main.c:380]:
```suggestion
        _("Override projection check (use current project's CRS)");
```

File: vector/v.in.ogr/main.c [hunk 3]
```diff
@@ -377,9 +377,9 @@ int main(int argc, char *argv[])
...
     flag.over->description = _("Assume that the dataset has the same "
-                               "projection as the current location");
+                               "projection as the current project");
```
Comment (veroandreo) [inline vector/v.in.ogr/main.c:382]:
```suggestion
                               "coordinate reference system (CRS) as the current project");
```

File: vector/v.in.ogr/proj.c [hunk 4]
```diff
@@ -490,10 +490,10 @@ void check_projection(struct Cell_head *cellhd, GDALDatasetH hDS, int layer,
                 strcat(error_msg, _("\nIn case of no significant differences "
                                     "in the projection definitions,"
```
Comment (veroandreo) [inline vector/v.in.ogr/proj.c:491]: Perhaps we want to change projection by CRS

File: vector/v.in.ogr/proj.c [hunk 5]
```diff
@@ -511,7 +511,7 @@ void check_projection(struct Cell_head *cellhd, GDALDatasetH hDS, int layer,
                 msg_fn = G_message;
             else
                 msg_fn = G_verbose_message;
-            msg_fn(_("Projection of input dataset and current location "
+            msg_fn(_("Projection of input dataset and current project "
```
Comment (veroandreo) [inline vector/v.in.ogr/proj.c:514]: Same as above

**Review by veroandreo — COMMENTED** [review]
I'd consider editing at least those messages/sentences that include projection and project... Sounds repetitive and IMO, reference system or CRS is more appropriate. What do you think?

**Comment by nilason** [comment]
> I'd consider editing at least those messages/sentences that include projection and project... Sounds repetitive and IMO, reference system or CRS is more appropriate. What do you think?

You’re right, but perhaps an issue for another PR…?

**Comment by petrasovaa** [comment]
> > I'd consider editing at least those messages/sentences that include projection and project... Sounds repetitive and IMO, reference system or CRS is more appropriate. What do you think?
> 
> You’re right, but perhaps an issue for another PR…?

I mostly agree, and I tried to do it in several places in manual pages, but I admit I soon got tired of it. Also I was unsure whether CRS is well known, so I tried using "coordinate reference system (CRS)" which is kind of long.

File: vector/v.in.ogr/proj.c [hunk 6]
```diff
@@ -365,13 +365,13 @@ void check_projection(struct Cell_head *cellhd, GDALDatasetH hDS, int layer,
             int i_value;
 
             strcpy(error_msg, _("Projection of dataset does not"
```
Comment (veroandreo) [inline vector/v.in.ogr/proj.c:367]:
```suggestion
            strcpy(error_msg, _("Coordinate reference system of dataset does not"
```

**Review by veroandreo — APPROVED** [review]

**Review by veroandreo — APPROVED** [review]

**Pull request merged**